Reshma uva

Reshma27Menon opened the pull request:
Added personal profile markdown and photo for IRIS-HEP website.

Reshma27Menon committed:
My intro slide

Reshma27Menon committed:
Updated .md

Reshma27Menon committed:
Added my picture

Reshma27Menon committed:
Created a bio

pre-commit-ci[bot] committed:
style: pre-commit fixes

rct225 commented:
@Reshma27Menon please resize your photo in this PR -It should be 320x240 pixels and in general should be 150kB or less.

Reshma27Menon committed:
Updated pic dimensions

Reshma27Menon committed:
Important
Merge branch 'reshma_uva' of github.com:Reshma27Menon/iris-hep.github.io into reshma_uva
